x/nodes/keeper/reward.go

```diff
@@ -7,13 +7,53 @@ import (
 	sdk "github.com/pokt-network/pocket-core/types"
 	govTypes "github.com/pokt-network/pocket-core/x/gov/types"
 	"github.com/pokt-network/pocket-core/x/nodes/types"
+	pcTypes "github.com/pokt-network/pocket-core/x/pocketcore/types"
 )
 
 // RewardForRelays - Award coins to an address using the default multiplier
 func (k Keeper) RewardForRelays(ctx sdk.Ctx, relays sdk.BigInt, address sdk.Address) sdk.BigInt {
 	return k.RewardForRelaysPerChain(ctx, "", relays, address)
 }
 
+// CalculateRelayReward - Calculate the amount of rewards based on the given
+// number of relays and the staked tokens.  The returned reward amount includes
+// DAO & Proposer portions.
+func (k Keeper) CalculateRelayReward(
+	ctx sdk.Ctx,
+	chain string,
+	relays sdk.BigInt,
+	stake sdk.BigInt,
+) (sdk.BigInt, sdk.BigInt) {
```

Comment:
Can you named the return params so it's clear if it is `(daoReward, proposerReward)` or vice versa?